client/Wayland/wlf_input.c

```diff
@@ -441,7 +444,11 @@ BOOL wlf_handle_touch_up(freerdp* instance, const UwacTouchUp* ev)
 		return FALSE;
 
 	WINPR_ASSERT(rdpei->TouchEnd);
+	int contactId;
 	rdpei->TouchEnd(rdpei, touchId, x, y, &contactId);
+#else
+	WLog_WARN(TAG, "Touch event detected but RDPEI support not compiled in");
+#endif
```

Comment:
Possibly we should add hint how to disable rdpei.

Reply:
you mean enable? added that to the log.